ddl/index.go

```diff
@@ -99,7 +99,7 @@ func checkPKOnGeneratedColumn(tblInfo *model.TableInfo, indexPartSpecifications
 		}
 		// Virtual columns cannot be used in primary key.
 		if lastCol.IsGenerated() && !lastCol.GeneratedStored {
-			return nil, ErrUnsupportedOnGeneratedColumn.GenWithStackByArgs("Defining a virtual generated column as primary key")
```

Comment:
could you add some tests?

Reply:
i think it is not simply change one line

Reply:
I think the manual test is enough.